src/shared/sam/debugger/awsSamDebugger.ts

```diff
@@ -331,18 +331,6 @@ export class SamDebugConfigProvider implements vscode.DebugConfigurationProvider
             vscode.Uri.parse(templateInvoke.templatePath!!)
 
         let awsCredentials: Credentials | undefined
-
-        // TODO: Remove this line to enable dotnetcore3.1 debugging when it becomes available
-        if (runtime === 'dotnetcore3.1' && !config.noDebug) {
```

Comment:
In the past, we've gated functionality around the detected SAM CLI versions when we didn't raise the minimum supported version.

Reply:
Do we have an easy way to gate this? Raising min for everyone seems a little aggressive

Reply:
I agree that we wouldn't want to raise the min version
#785 has an example of version check gating

Reply:
What would you have it do? Fail?

Reply:
Maybe behave the same way as this but update the message with the minver for dotnetcore3.1 debugging?

Reply:
Added